api/providers.go

```diff
@@ -83,6 +85,11 @@ func loadExternalProviders(endpointsConfgFiles []string) (externalProviders LogP
 }
 
 func loadSystemdCollectors(cfg *config.Config, DCOSTools dcos.Tooler) ([]collector.Collector, error) {
+	if runtime.GOOS == "windows" {
```

Comment:
Ideally we wold not even have `func ReadJournalOutputSince(ctx context.Context, unit string, duration time.Duration) (io.ReadCloser, error)` in `helpers_windows.go`. However, the build fails if I remove them.